SU-SWS/decanter
# Cut down on media queries generated by @modular-spacing

# READY FOR REVIEW

# Summary
- @modular-spacing use a for loop to generate media queries for every breakpoint so total of 6 queries per element we use the mixin on.
- `xs` and `sm` shared the same base value, `md, lg, xl` shared another one, so we really only need 3 media queries for each element that uses @modular-spacing
- Propose one more variable map for media queries for use with modular-spacing so we can cut the number of queries from 6 to 3 for every use.

# Needed By (Date)
- Whenever

# Urgency
- Not Urgent

# Steps to Test

Look at the "Files changed" tabs here on this GitHub page. You should be able to see in decanter.css the deleted media queries that are not really needed.

OR

1. Pull this branch and run grunt styleguide
2. Look at .su-global-footer in decanter.css - it should now only have 3 media queries for the padding-top and padding-bottom instead of 6.

# Affected Projects or Products
- Decanter
